app/lib/pre_assembly/digital_object.rb

```diff
@@ -149,28 +148,14 @@ def current_object_version
       @current_object_version ||= version_client.current.to_i
     end
 
-    # When reaccessioning, we need to first open and close a version without kicking off accessionWF
-    def create_new_version
-      version_client.open(
-        significance: 'major',
-        description: 'pre-assembly re-accession',
-        opening_user_name: bundle.bundle_context.user.sunet_id
-      )
-      version_client.close(start_accession: false)
-    end
-
-    ####
-    # Initialize the assembly workflow.
-    ####
-
-    # Call web service to add assemblyWF to the object in DOR.
-    def initialize_assembly_workflow
-      workflow_client.create_workflow_by_name(druid.druid, 'assemblyWF', version: current_object_version)
-    end
-
-    def workflow_client
-      logger = Logger.new(Settings.workflow.logfile, Settings.workflow.shift_age)
-      Dor::Workflow::Client.new(url: Settings.workflow_url, logger: logger, timeout: Settings.workflow.timeout)
+    def start_accession
+      version_params =
+        {
+          significance: 'major',
+          description: 'pre-assembly re-accession',
+          opening_user_name: bundle.bundle_context.user.sunet_id
+        }
+      object_client.accession.start(version_params)
```

Comment:
Related: I wonder if this call will work without this line:

https://github.com/sul-dlss/dor-services-client/blob/master/lib/dor/services/client/accession.rb#L28

Changing to `req.params = params.to_json` ?

Reply:
huh, i was assuming that faraday would take care of the serialization - certainly the testing in stage will reveal if this works or not

Reply:
@peetucket yeah, let's find out. it may not be needed